exampleCourse/questions/stringOperations/question.html

```diff
@@ -0,0 +1,66 @@
+<pl-question-panel>
+  <p> 1) For the following python code snippet, enter below the resulting string:</p>
+
+<pl-code language="python">
+a = "{{params.stringname}}"*{{params.a}}
+</pl-code>
+
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans1"  label = "a = ">
+</pl-string-input>
+</p>
+
+<pl-question-panel>
+<p> 2) For the following python code snippet, what is d = b + c?</p>
+<pl-code language="python">
+  b = "{{params.b}}"
+  c = "{{params.c}}"
+</pl-code>
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans2" label = "b + c = " >
+</pl-string-input>
+</p>
+
+<pl-question-panel>
+<p>3) Enter the string below, however include trailing and leading blanks.
+ When using the attribute <code>remove_leading_trailing=True</code>, the answer will
```

Comment:
Should be `remove_leading_trailing="true"`, I think?

This also occurs in more places below (in both the actual HTML and in the descriptive text).